clang/test/Driver/clang-offload-bundler-bc-archive-support.cpp

```diff
@@ -0,0 +1,121 @@
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// Make bundled object with targets:

```

Comment:
I do not think this test will work on windows...as there is a linux specific command 'rm'. Can you please check?

Thanks

Reply:
Test m-and-mm.c also uses rm.